packages/swingset/src/resolvers/component.ts

```diff
@@ -80,6 +82,7 @@ export async function resolveComponents({
         relativePath,
         slug: componentSlug,
         title: (frontmatter.title as string) ?? componentSlug,
+        parsedPath: parseComponentPath(componentPath)
```

Comment:
Instead of `componentPath`, we'll want to use `frontmatter.path` here. The path that users can define in frontmatter is independent of a component's filepath 👍